cmake/Summary.cmake

```
@@ -98,8 +98,8 @@ endif()
     message(STATUS "  BUILD_HIPSTDPAR_TEST            : ${BUILD_HIPSTDPAR_TEST}")
     message(STATUS "  BUILD_HIPSTDPAR_TEST_WITH_TBB   : ${BUILD_HIPSTDPAR_TEST_WITH_TBB}")
     message(STATUS "  BUILD_EXAMPLES                  : ${BUILD_EXAMPLES}")
-    message(STATUS "  BUILD_BENCHMARKS                : ${BUILD_BENCHMARKS}")
-if(BUILD_BENCHMARKS)
+    message(STATUS "  BUILD_BENCHMARK                : ${BUILD_BENCHMARK}")
```

For alignment with the surrounding lines:
```suggestion
    message(STATUS "  BUILD_BENCHMARK                 : ${BUILD_BENCHMARK}")
```